### PR TITLE
Add CoreWeave to adopters

### PR DESCRIPTION
**Pull request by salanki**
**What this PR does / why we need it**:
Adds [CoreWeave](https://www.coreweave.com) to adopters.

We built our cloud to be Kubernetes on bare-metal and had initially no plans to support hypervisor based workloads. The pandemic drew a large demand for Virtual Desktops and Kubevirt allowed us to deploy a VM product in a short period of time integrating smoothly in our existing stack. Thank you.

### CODE REVIEW

**Comment by rmohr** [comment]
/release-note-none

@salanki would you mind signing the commit to make the `dco` check pass?

File: ADOPTERS.md [hunk 1]
```diff
@@ -6,4 +6,5 @@ This is a likely incomplete list of KubeVirt adopters - end-users and distributo
...
-| H3C | 2019 | We distribute KubeVirt as part of CloudOS to enable VM workloads on Kubernetes at customer sites . Follow the  [link](https://www.h3c.com/en/Products_Technology/Enterprise_Products/Cloud_Computing/Cloud_Computing_Products/H3C_CloudOS/H3C_CloudOS_full-stack/) for more product information |
\ No newline at end of file
+| H3C | 2019 | We distribute KubeVirt as part of CloudOS to enable VM workloads on Kubernetes at customer sites . Follow the  [link](https://www.h3c.com/en/Products_Technology/Enterprise_Products/Cloud_Computing/Cloud_Computing_Products/H3C_CloudOS/H3C_CloudOS_full-stack/) for more product information |
+| [CoreWeave](https://www.coreweave.com) | 2020 | A Kubernetes native cloud provider with focus on GPUs at scale. Kubevirt allows us to co-locate non-containerizable workloads such as Virtual Desktops next to compute intensive containers executing on bare metal. All orchestrated via the Kubernetes API leveraging the same network policies and persistent volumes for both VM and containerized workloads. |
```
Comment (fabiand) [inline ADOPTERS.md:10]: s/Kubevirt/KubeVirt/

**Review by davidvossel — APPROVED** [review]
/approve
/lgtm

Excellent to hear KubeVirt being used for VDI

**Comment by davidvossel** [comment]
/lgtm cancel 

need doc and the Kubevirt vs KubeVirt capitalization fix

File: ADOPTERS.md (hunk 1 above)
Comment (salanki) [reply ADOPTERS.md]: Fixed

**Comment by salanki** [comment]
Fixed

**Review by davidvossel — APPROVED** [review]
/lgtm

**Comment by salanki** [comment]
Merge please

**Pull request merged**